docs/tails_printing_guide.rst

```diff
@@ -7,12 +7,75 @@ potentially untrusted network. This is why **we strongly recommend using
 a printer that does not have WiFi or Bluetooth**, and hooking up to it
 using a regular USB cable to print.
 
-Normally, any printer should work with Tails "out of the box." Most
-difficulties stem from not selecting the right driver (extra software
-needed for the printer and computer to communicate). Luckily, Tails has
-a large number of drivers for just about any popularly manufactured
-printer on hand, without even having to download new drivers from the
-web.
+Finding a printer that works with Tails can be challenging because Tails is
+based on the Linux operating system, which often has second-class hardware
+support in comparison to operating systems such as Windows or macOS.
+
+We `maintain a list of printers
+<https://docs.securedrop.org/en/stable/hardware.html#printers>`_ that we have
```

Comment:
After some research, I've used this convention, per the official Sphinx docs on `:ref`:

Here are the steps:

* Add the reference point `.. _tails_printing_guide:` above the section titled `Setting up a Printer in Tails` in `tails_printing_guide.rst`.

* In `hardware.rst`, link to this reference point by including `` :ref:`tails_printing_guide` `` within the paragraph text, which produces a link that inherits the title of the section being referred to. (One can customize the link text, but leaving it as-is seemed appropriate here.)

Here's the result:

![ref](https://user-images.githubusercontent.com/2104294/28392396-9d306158-6c96-11e7-8bc8-962de9fa4992.png)

This passes linting and, as you pointed out, ensures that developers running a local copy will *only* be pointed to local copies of files; if this method is good etiquette, I'll employ it going forward. That, and make note of it for consideration when we further discuss revisions to the style guide.